.goreleaser.yaml

```diff
@@ -0,0 +1,129 @@
+# run a local "snapshot" build without publishing:
+#    TRAVIS_BUILD_NUMBER=1 goreleaser --snapshot --rm-dist --debug
+#
+project_name: chezmoi
+
+builds:
+- binary: chezmoi
+  env:
+    - CGO_ENABLED=0
+  ldflags:
+     - -s -w
+  goos:
+    - linux
+    - darwin
+    # TODO: uncomment when compatible with windows. currently fails on 'undefined: syscall.Stat_t'
```

Comment:
I noticed I couldn't compile on windows at the moment. If that's ever supported you can comment out this line and you'll start generating windows/x86-64 releases